### ISSUE TITLE
Strict mode error with directive $timeout

### ISSUE THREAD

**Issue by LiamK** [mareczek/international-phone-number#34]
Hi, Mareczek,

Thanks for the response -- the ng-required now works!  I'm using 0.0.6.  I ran into one small error which I hope you can fix.  I get an insertion error which prevents it from working.  I made the following simple modification in the releases/international-phone-number.js file.  I am not familiar with Coffeescript.

Current version:
```
(function() {
  "use strict";
  angular.module("internationalPhoneNumber", []).directive('internationalPhoneNumber', function($timeout) { ... });
```
My fixed version:

```
(function() {
  "use strict";
  angular.module("internationalPhoneNumber", []).directive('internationalPhoneNumber', ['$timeout', function($timeout) { ... }]);
```

Thanks for your help!

Liam

**Comment by LiamK** [comment]
I'm referring to strict-di mode, of course.

**Comment by mareczek** [comment]
This issue was resolved in 0.0.8 (i think), for sure it's resolved in 0.0.9 (the one that i deployed today).

**Comment by mareczek** [comment]
Please drop a line in case you experience any problems
